### PR TITLE
BUG: Remove SpatialObjectProperty's writable string methods from SWIG

### LINKED ISSUE #4779: Wrapping of member functions that are passed and/or return std:string goes wonky
The issues are demonstrated in https://github.com/InsightSoftwareConsortium/ITK/pull/4776

1) SWIG seems to be converting some member functions that are passed two std::strings as args to new versions of functions that returns a std::string * (rather than returning a void as defined by the C++ code). 

This documentation, though old, might be relevant...
https://www.swig.org/Doc1.3/Arguments.html#Arguments_nn2

That documented feature would cause a function in C++ that returns a void to wrap to a python function that returns a value - hence causing the SetTagValueString to appear to return a value in python.

2) Additionally,  SWIG converts return by value to return by pointer:
https://www.swig.org/Doc1.3/SWIG.html#SWIG_nn23 

So,  that is where the std::string * references are coming from.

**Summary**

I suspect there is a bug in our SWIG wrapping specification for these reasons and because new SWIG should handled std::string fine and these wonkynesses shouldn't be happening.

### PR DESCRIPTION
**Pull request by blowekamp**
SWIG's binding for writable std::string objects is a "std::string *" which is not implemented.

closes #4779 

 replaces #4776

## PR Checklist
- [ ] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [ ] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [ ] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [ ] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [ ] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

**Comment by blowekamp** [comment]
Seems to work:
image

**Review by dzenanz — COMMENTED** [review]
LGTM, but I defer to @aylward

**Review by aylward — APPROVED** [review]

File: Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h [hunk 1]
```diff
@@ -123,15 +125,13 @@ class ITKSpatialObjects_EXPORT SpatialObjectProperty
     return value;
   }
 
+
+#if !defined(SWIG) && !defined(ITK_WRAPPING_PARSER)
```
Comment (blowekamp) [inline Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h:129]: I think this should only be dependent on ITK_WRAPPING_PARSER

**Comment by blowekamp** [comment]
@aylward @thewtex  Does this fix need to be applied to the release branch too?

**Comment by thewtex** [comment]
@blowekamp thanks for following up with a patch on this!

> @aylward @thewtex Does this fix need to be applied to the release branch too?

Yes, done! https://github.com/InsightSoftwareConsortium/ITK/commit/efe773f555a567079b07a47eb769d10ff7b56671

**Pull request merged**